### PR TITLE
Echo 87 fix ft orders

### PR DESCRIPTION
**Pull request by f-pto**

### CODE REVIEW

File: packages/sdk/order/approve.ts [hunk 1]
```diff
@@ -25,23 +25,28 @@ export async function approve_fa1_2_arg(
...
-  infinite: boolean = true,
   spender?: string
 ) : Promise<TransactionArg | undefined > {
   spender = spender || provider.config.transfer_proxy
   const st : StorageFA1_2 = await provider.tezos.storage(contract)
   let key_exists = false
   let balance = undefined
   try {
-    let r : any = await st.ledger.get(owner)
+    let r : any = await st.allowance.get(
```
Comment (rzcoder) [inline packages/sdk/order/approve.ts:35]: This didn't build

`Property 'allowance' does not exist on type 'StorageFA1_2'.`

**Review by rzcoder — CHANGES_REQUESTED** [review]
Please resolve compiling error

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
[the remark this answers is not in the document]
Comment (f-pto) [reply .vscode/launch.json]: Fixed !

File: packages/sdk/order/approve.ts (hunk 1 above)
Comment (f-pto) [reply packages/sdk/order/approve.ts:35]: Fixed !

**Pull request merged**